examples/typescript/aws-cloudfront-proxy/__tests__/main-test.ts

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) HashiCorp, Inc
+// Licensed under the Mozilla Public License 2.0
```

Comment:
the RFC about this requirement was updated and now requires the following it seems:
```suggestion
// SPDX-License-Identifier: MPL-2.0
```